server/src/main/java/com/linecorp/centraldogma/server/internal/api/ContentServiceV1.java

```diff
@@ -368,4 +369,13 @@ private static Object handleWatchFailure(Throwable thrown) {
         }
         return converter.apply(Iterables.getOnlyElement(collection));
     }
+
+    @Get("/projects/{projectName}/repos/{repoName}/aggregation")
```

Comment:
`aggregated` or `aggregate`? (because we are retrieving an aggregated entry)

Reply:
I like `aggregated`. Thank you~!